lib/rubocop/target_finder.rb

```diff
@@ -70,14 +72,19 @@ def find_files(base_dir, flags)
 
     private
 
-    def to_inspect?(file, hidden_files, base_dir_config)
+    # rubocop:disable Metrics/CyclomaticComplexity
+    def to_inspect?(file, hidden_files, base_dir, base_dir_config)
       return false if base_dir_config.file_to_exclude?(file)
       return true if !hidden_files.bsearch do |hidden_file|
         file <=> hidden_file
       end && ruby_file?(file)
+      return true if base_dir.include?(HIDDEN_PATH_SUBSTRING) &&
+                     file.start_with?(base_dir) &&
```

Comment:
I've found that lines 2 and 3 of the condition aren't covered by tests. The specs still pass if I remove `file.start_with?(base_dir)` or `ruby_file?(file)`  from the condition.

Reply:
`file.start_with?(base_dir)` is now obsolete; it was removed. Altered specs to fail without the `ruby_file?(file)` condition.